README.md

```diff
@@ -1,229 +1,40 @@
-# Starter kit for a Terraform GitOps module
+# Cloud Pak for Security GitOps module
 
-This is a Starter kit to help with the creation of Terraform modules. The basic structure of a Terraform module is fairly
-simple and consists of the following basic values:
-
-- README.md - provides a description of the module
-- main.tf - defines the logic for the module
-- variables.tf (optional) - defines the input variables for the module
-- outputs.tf (optional) - defines the values that are output from the module
-
-Beyond those files, any other content can be added and organized however you see fit. For example, you can add a `scripts/` directory
-that contains shell scripts executed by a `local-exec` `null_resource` in the terraform module. The contents will depend on what your
-module does and how it does it.
-
-## Instructions for creating a new module
-
-1. Update the title and description in the README to match the module you are creating
-2. Fill out the remaining sections in the README template as appropriate
-3. Implement your logic in the in the main.tf, variables.tf, and outputs.tf
-4. Use releases/tags to manage release versions of your module
+Module to populate a gitops repository with the resources to deploy IBM Cloud Pak for Security. 
```

Comment:
Do we need more information to explain what is being installed?  Does it install all of CP4s? (It doesn't look like it).  Is there anything else I need to use this?

Reply:
Yes @triceam This installs all of CP4S Ref: https://www.ibm.com/docs/en/cloud-paks/cp-security/1.9?topic=installing-cloud-pak-security-by-using-openshift-cli

Reply:
> @dimallya That's the approach we've been taking for all other offerings. We have have a module that installs the operator and deploys the cloud pak instance. In some cases these are separated out into separate modules, in other cases they are together. Then, we have a separate module for each software products within the cloud pak. This lets us generate terraform automation that does a full install for the specific software products that we want.
> 
> For example, for CP4D we can automate deployment of specific products like DB2 or WKC, which will automatically include their dependencies.
> 
> The intent of the Software Everywhere effort is to extend this level of modularity for all Cloud Pak offerings. So, for CP4S, we could have individual modules that install QRadar, Security Guardium, Security Data Explorer, etc... Each would have a dependency on the core module that installs the CP4S operator/instance.
> 
> So, just to make sure I understand clearly, this module installs the CP4S operator and creates the CP4S (`CP4SThreatManagement`) instance. Correct?

Yes @triceam That's right. This module installs the CP4S operator and creates the CP4S (`CP4SThreatManagement`) instance.

I understand and agree with your points regarding SWE modularity for Cloud Paks. As far as this module is concerned do we have need to have separate module for operator and instance of CP4s?